public/index.js

```diff
@@ -38,12 +38,36 @@ function volumeDown() {
   }
 }
 
+// positions of insectNet and waterGun
+let insectNetActive = false;
+let waterGunActive = false;
+
 function toggleInsectNet() {
-  const icon = document.querySelector(".tools0");
-  if (icon.classList.contains("insectNet-active")) {
-      icon.classList.remove("insectNet-active");
+  waterGunActive = false;
+  insectNetActive = !insectNetActive;
+  updateToolIcons();
+}
+
+function toggleWaterGun() {
+  insectNetActive = false;
+  waterGunActive = !waterGunActive;
+  updateToolIcons();
+}
+
+function updateToolIcons() {
+  const insectNetIcon = document.querySelector(".tools0");
+  const waterGunIcon = document.querySelector(".tools1");
+  
+  if (insectNetActive) {
+    insectNetIcon.classList.add("insectNet-active");
   } else {
-      icon.classList.add("insectNet-active");
+    insectNetIcon.classList.remove("insectNet-active");
+  }
+  
+  if (waterGunActive) {
+    waterGunIcon.classList.add("waterGun-active");
+  } else {
+    waterGunIcon.classList.remove("waterGun-active");
   }
```

Comment:
```suggestion
  // TODO: 水鉄砲の機能が実装できたらコメントアウトをやめる
  // if (waterGunActive) {
  //  waterGunIcon.classList.add("waterGun-active");
  // } else {
  //   waterGunIcon.classList.remove("waterGun-active");
  // }
```
未使用のロジックは不要なエラーを発生させるため、コメントアウトなどで回避しましょう。